graph/src/blockchain/types.rs

```diff
@@ -186,7 +186,10 @@ impl TryFrom<(&[u8], i64)> for BlockPtr {
             H256::from_slice(bytes)
         } else {
             return Err(anyhow!(
-                "invalid H256 value `{}` has {} bytes instead of {}"
+                "invalid H256 value `{:?}` has {} bytes instead of {}",
+                bytes,
```

Comment:
Maybe we should print has hexadecimal using `hex::encode(bytes)`, that would render the actual message easier to read.